client/packages/programs/src/JsonForms/components/Prescription/Prescription.tsx

```diff
@@ -72,8 +72,16 @@ const UIComponent = (props: ControlProps) => {
     useState<ItemStockOnHandFragment | null>(
       formActions.getState(`${path}_item`) ?? null
     );
-  const { draftStockOutLines, setDraftStockOutLines } =
-    useDraftPrescriptionLines(selectedItem);
+
+  const [draftPrescriptionLines, setDraftPrescriptionLines] = useState<
+    DraftStockOutLine[]
+  >([]);
```

Comment:
Because of the changes to state management, this component now needs to hold its own `draftLines` state, and it gets manipulated by the `useDraftPrescriptionLines` hook -- functionality should be unchanged.